src/openloco/windows/construction/OverheadTab.cpp

```diff
@@ -0,0 +1,387 @@
+using namespace openloco::map::tilemgr;
+
+namespace openloco::ui::windows::construction::overhead
+{
+    widget_t widgets[] = {
+        commonWidgets(138, 192, string_ids::stringid_2),
+        make_widget({ 3, 45 }, { 132, 12 }, widget_type::checkbox, 1, string_ids::empty, string_ids::tooltip_select_track_mod),
+        make_widget({ 3, 57 }, { 132, 12 }, widget_type::checkbox, 1, string_ids::empty, string_ids::tooltip_select_track_mod),
+        make_widget({ 3, 69 }, { 132, 12 }, widget_type::checkbox, 1, string_ids::empty, string_ids::tooltip_select_track_mod),
+        make_widget({ 3, 81 }, { 132, 12 }, widget_type::checkbox, 1, string_ids::empty, string_ids::tooltip_select_track_mod),
+        make_widget({ 35, 110 }, { 66, 66 }, widget_type::wt_3, 1),
+        make_widget({ 3, 95 }, { 132, 12 }, widget_type::wt_18, 1, 0xFFFFFFFF, string_ids::tooltip_select_track_to_upgrade),
+        make_widget({ 123, 96 }, { 11, 10 }, widget_type::wt_11, 1, string_ids::dropdown, string_ids::tooltip_select_track_to_upgrade),
+        widget_end(),
+    };
+
+    window_event_list events;
+
+    // 0x0049EBD1
+    static void on_mouse_up(window* self, widget_index widgetIndex)
+    {
+        switch (widgetIndex)
+        {
+            case common::widx::close_button:
+                WindowManager::close(self);
+                break;
+
+            case common::widx::tab_construction:
+            case common::widx::tab_overhead:
+            case common::widx::tab_signal:
+            case common::widx::tab_station:
+                common::switchTab(self, widgetIndex);
+                break;
+
+            case widx::checkbox_1:
+            case widx::checkbox_2:
+            case widx::checkbox_3:
+            case widx::checkbox_4:
+            {
+                auto checkboxIndex = widgetIndex - 8;
+
+                if (_lastSelectedMods & 1 << checkboxIndex)
+                    _lastSelectedMods = _lastSelectedMods & ~(1 << checkboxIndex);
+                else
+                    _lastSelectedMods = _lastSelectedMods | (1 << checkboxIndex);
+
+                // TODO: & ~(1 << 7) added to prevent crashing when selecting/deselecting overhead wires for trams
+                _scenarioTrackMods[_trackType & ~(1 << 7)] = _lastSelectedMods;
+
+                self->invalidate();
+                break;
+            }
+        }
+    }
+
+    // 0x0049EBFC
+    static void on_mouse_down(window* self, widget_index widgetIndex)
+    {
+        switch (widgetIndex)
+        {
+            case widx::track_dropdown:
+            {
+                uint8_t modCount = 3;
+
+                auto widget = self->widgets[widx::track];
+                auto xPos = widget.left + self->x;
+                auto yPos = widget.top + self->y;
+                auto width = widget.width() + 2;
+                auto height = widget.height();
+
+                dropdown::show(xPos, yPos, width, height, self->colours[1], modCount, 0x80);
```

Comment:
0x80 is a flag represent it as `(1 << 7)` and indicates that the widget should be marked as depressed i think.